src/ctest.c

```diff
@@ -316,12 +316,12 @@ void int16_t_ToString(char* string, size_t bufferSize, int16_t val)
 
 void uint32_t_ToString(char* string, size_t bufferSize, uint32_t val)
 {
-    (void)snprintf(string, bufferSize, "%llu", (unsigned long long)val);
```

Comment:
>erSize, "%llu", (unsigned long long)val); [](start = 31, length = 41)

Actually the correct fix here would be to use PRIu32.
Same for the others in this file. I do not remember why this was not done when the file was written, but there should be no reason for it, as we already use the fixed size types all over in this file.

Reply:
Never worked with PRI macros before -- this looks much better.

Reply:
Awesome, thanks, I'll pull it in.

---
In reply to: [192471220](https://github.com/Azure/azure-ctest/pull/9#discussion_r192471220) [](ancestors = 192471220)